vprdb/converters/kitti_converter.py

```diff
@@ -0,0 +1,71 @@
+import open3d as o3d
+import pykitti
+import numpy as np
+import mrob
+
+
+def kitti_to_lib_format(
+    source_dir: str,
+    sequence: str,
+    camera: int,
+    output_pcd_dir: str,
+    output_poses_dir: str,
```

Comment:
Let's save output to one directory for easy reading with our instrument. Therefore, one argument is needed here instead of two